### PR TITLE
chore: upgrade to typescript-eslint v8 final

### PR DESCRIPTION
**Pull request by benmccann**
did I do the changeset correctly by including only the one package?

### CODE REVIEW

File: .changeset/eleven-ways-fry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@svelte-add/adders': patch
```
Comment (AdrianGonz97) [inline .changeset/eleven-ways-fry.md:2]:
```suggestion
'@svelte-add/adders': patch
'svelte-add': patch
```

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
 		"rollup-plugin-preserve-shebangs": "^0.2.0",
 		"svelte-add": "workspace:*",
 		"typescript": "^5.5.2",
-		"typescript-eslint": "8.0.0-alpha.37"
+		"typescript-eslint": "8.0.0"
```
Comment (AdrianGonz97) [inline package.json:45]:
```suggestion
		"typescript-eslint": "^8.0.0"
```

**Review by AdrianGonz97 — COMMENTED** [review]
almost! `@svelte-add/adders` is the special case where `svelte-add` also needs to be included in the changeset

however, I think we should probably add `@svelte-add/adders` to the changeset ignore list too since its presence is more confusing than helpful. I don't see a point in maintaining a separate changelog for it when we're always going to need to include `svelte-add` with it anyway

**Review by AdrianGonz97 — APPROVED** [review]

**Comment by manuel3108** [comment]
> however, I think we should probably add `@svelte-add/adders` to the changeset ignore list too since its presence is more confusing than helpful. I don't see a point in maintaining a separate changelog for it when we're always going to need to include `svelte-add` with it anyway

Yeah, I think that makes sense. Apart from that everything looks good.

**Pull request merged**